ocs_ci/utility/yaml_log_filter.py

```
@@ -0,0 +1,118 @@
+    kind = data.get("kind", "Unknown")
+
+    if kind == "List":
+        items = data.get("items", [])
+        item_kinds: dict[str, int] = {}
+        for item in items:
+            k = item.get("kind", "Unknown")
+            item_kinds[k] = item_kinds.get(k, 0) + 1
+
+        summary = ", ".join(f"{count} {k}" for k, count in item_kinds.items())
+        return f"[List: {summary}] (filtered from {original_size:,} chars)"
+    else:
```

Ruff (0.15.21)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)